### ISSUE TITLE
Generating html code coverage fails when using some next generation features

### ISSUE THREAD

**Issue by queicherius** [queicherius/abc-environment#1]
So it turns out that the test coverage fails if
- ... you are using an `if` statement
- ... you are using default parameters in the function

It seems to be a weird combination of the `babel-plugin-rewire` and `babel-plugin-istanbul`.

---

``` js
// Modul
function (something, punctuation = '!') {
  if (something === 'test') {
    console.log('Lol')
  }

  return answer + punctuation
}
```

---

``` js
// Test
import {expect} from 'chai'
import module from '../src/xd.js'

describe('my module', () => {
  it('should return the answer to life', async () => {
    let result = await module('hello')
    expect(result).to.equal('42!')
  })

  it('should return the answer to life', async () => {
    let result = await module('test', '?')
    expect(result).to.equal('42?')
  })
})
```

**Comment by queicherius** [comment]
See [this issue](https://github.com/speedskater/babel-plugin-rewire/issues/165) and [this pull request](https://github.com/istanbuljs/istanbul-reports/pull/11)

**Comment by queicherius** [comment]
Fixed with 82cbcdbec0ecb513955f4c6405c87c817fc23af3
